debugger/src/main/java/org/codice/acdebugger/api/DomainUtil.java

```diff
@@ -71,23 +71,14 @@ public String get(StackFrame frame) {
    */
   @Nullable
   public String get(@Nullable Object obj) {
-    if (obj instanceof StackFrame) {
-      obj = ((StackFrame) obj).location().declaringType().classObject();
-    } else if (obj instanceof ReferenceType) {
-      obj = ((ReferenceType) obj).classObject();
-    }
     if (obj == null) {
       return null;
     }
     final Map<Object, String> cache =
         debug.computeIfAbsent(DomainUtil.DOMAIN_LOCATION_CACHE, ConcurrentHashMap::new);
+    final String location = get0(debug.reflection(), obj, cache);
```

Comment:
ℹ️ The code was re-arranged to line up with the bundle util logic making it much easier to follow and maintain